README.md

```diff
@@ -63,6 +63,11 @@ After you have set the HTML and CSS, you can call the `emogrify` method to merge
 
     $mergedHtml = $emogrifier->emogrify();
 
+For Emogrifier to properly deal with entities and UTF-8 characters you need to add Content-Type meta tag with UTF-8 charset
+declaration to your HTML. If you don't provide this tag in your HTML then Emogrifier adds it automatically.
```

Comment:
Could you please reformat the lines in the README to be under 80 characters? This will avoid wrapping on GitHub.